server/db/db.js

```diff
@@ -0,0 +1,13 @@
+const { Pool } = require('pg');
+const fs = require('fs');
+require("dotenv").config();
+
+const db = new Pool({
+    host: 'localhost',
+    port: process.env.PGPORT,
+    database: 'products',
```

Comment:
I would update this for consistency 
```suggestion
    database: process.env.DB,
```